docker/docker-compose.yml

```diff
@@ -4,15 +4,15 @@ services:
   # Mainflux services
   users:
     image: mainflux/users:0.10.0
-    env_file: '../../stack/env.d/mainflux-users.env'
+    env_file: './env.d/mainflux-users.env'
```

Comment:
As we discussed before, instead of creating another commit to fix that, you can do this change in this commit _8bf47ab687611f1c02bdada197a4e0eb684eec5c_ directly.

Reply:
Done.

Reply:
Awesome, thanks!